_layouts/page.html

```diff
@@ -1,12 +1,31 @@
+
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en">
   <head>
     <title>{{site.title}}</title>
     <link href="{{ '/static/site.css' | relative_url }}" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" type="text/css" href="{{site.baseurl}}/assets/css/glossary.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
   </head>
-  <body>
+  <body style="width: 100%">
```

Comment:
This is now taken care of by the CSS

```suggestion
  <body>
```

Reply:
I thought I'd managed to remove all of these! Argh....